### DISCUSSION TITLE
Planning Mode

### DISCUSSION THREAD

**Discussion by jdham89** [voxeltycoon/issues#490] [General] [answered]
It would be super helpful to have a planning mode or a blueprint mode where you could plan out your placement of buildings first before paying the full price for them. In this mode, if it would be possible to see the area of effect as well would be so very helpful. Even in the real world, there is a lot of planning that goes into proper placement and location of a factory or warehouse before it is built. This would be a great way to have that planning completed without spending all of your money just to find out that something needs to be moved a couple of tiles to the left or right of the original position.

**Comment by victorleblais** [comment, marked as answer]
A planning mode is planned for stage 2 of Steam Early Access: https://trello.com/c/fiMnQAh0/28-planning-mode

Suggestions about what the planning mode can do and how are always welcome though !
